autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -452,8 +460,9 @@ def fit(train_data,
 
         logger.setLevel(verbosity2loglevel(verbosity))
         allowed_kwarg_names = {
-            'feature_generator_type',
-            'feature_generator_kwargs',
+            'feature_generator',
+            'feature_generator_type',  # TODO: Remove on 0.1.0 release
+            'feature_generator_kwargs',  # TODO: Remove on 0.1.0 release
```

Comment:
should we start printing warning whenever to-be-deprecated args are encountered?